sdk/storage/azure-storage-common/inc/azure/storage/common/internal/storage_bearer_token_authentication_policy.hpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <azure/core/http/policies/policy.hpp>
+
+#include <shared_mutex>
```

Comment:
Technically, `std::unique_lock` is defined in `<mutex>`, and including `<shared_mutex>` alone does not have to make it available, so it is better to explicitly include both headers.
```suggestion
#include <mutex>
#include <shared_mutex>
```